e2e/fragments/caseDocuments.js

```diff
@@ -25,6 +25,30 @@ module.exports = {
     };
   },
 
+  uploadDocumentsWhenSWETNotRequired() {
```

Comment:
do we need to upload all documents again? It will just slow test, can we just update SWET document?

Reply:
As discussed, the page has validation for all the other fields. So, uploading everything